dialect/StablehloOps.td

```diff
@@ -506,6 +645,13 @@ def StableHLO_AddOp : StableHLO_BinaryElementwiseOp<"add",
 
     See
     https://www.tensorflow.org/xla/operation_semantics#element-wise_binary_arithmetic_operations.
+
+    Example:
+
+    ```mlir
+    %0 = stablehlo.add %arg0, %arg1 : tensor<4xf32>
+    %1 = stablehlo.add %arg2, %arg3 : (tensor<4xf32, #CSR>, tensor<4xf32, #DCSR>) -> tensor<4xf32, #CSR>
```

Comment:
Not entirely. Wanted to show it somewhere, and figured `add` was a fairly common op so would be a good location. Lets discuss this further.

Reply:
Sounds good - this doesn't have to be a blocker. In the future, we could discuss the overall principles behind documentation and tests, so that they are comprehensive by construction.